src/advanced_inv.cpp

```diff
@@ -859,7 +883,7 @@ void advanced_inventory::recalc_pane( side p )
     // add items from several locations.
     if( pane.area == AIM_ALL ) {
         auto &other = squares[panes[-p + 1].area];
-        auto &alls = squares[AIM_ALL];
+        auto &alls  = squares[AIM_ALL];
```

Comment:
Please don't insert extraneous whitespace to line things up, it really doesn't help, and causes a lot of busywork to keep things together.

The exception is if the stuff you're lining up is a table of values, then it's frequently worthwhile.